nixos/modules/virtualisation/containers.nix

```diff
@@ -45,23 +42,10 @@ in
       description = "Enable the OCI seccomp BPF hook";
     };
 
-    containersConf = mkOption {
-      default = {};
+    settings = mkOption {
```

Comment:
containersConf.settings ?

Reply:
I changed it to a top level option because I'm unsure of how to handle the option prefix issue I get when using `mkRemovedOptionModule`

Reply:
For some reason, `mkRemovedOptionModule` doesn't like renaming an option to its parent, e.g., `a.b.c` renamed to `a.b`.

Reply:
I'd prefer that we keep containersConf so it's unambiguous. I can take a look at the error later today.

Reply:
I reverted back to `settings`, but I can push a non working version.

Reply:
@zowoq Thanks for fixing the issue here.